packages/provider/src/language-model/v1/language-model-v1.ts

```diff
@@ -145,24 +145,6 @@ Finish reason.
       completionTokens: number;
     };
 
-    /**
-Raw prompt and setting information for observability provider integration.
-     */
-    // TODO v2: remove in v2 (now there is request)
```

Comment:
I assume that `"request"` is referring to the `.request` key:
https://github.com/vercel/ai/blob/56faf0dc29a00c936b101a8e30bc05e63fb4dab9/packages/provider/src/language-model/v1/language-model-v1.ts#L164-L173

But that key is marked as optional, while `.rawCall` seems to have always been present.

Maybe the `.request` key should not be optional?